tests/installation/bootloader_zkvm.pm

```diff
@@ -81,6 +86,7 @@ sub run {
 
     if (!get_var("BOOT_HDD_IMAGE") or (get_var('PATCHED_SYSTEM') and !get_var('ZDUP'))) {
         if (check_var("VIDEOMODE", "text")) {
+            return if (get_var('AGAMA'));
```

Comment:
again, why not in line 86 above? we will not find "Starting YaST ..." in the serial, the else will not be executed either, right? it is not related with being textmode or not.

Reply:
```
return if (get_var('AGAMA'));
    if (!get_var("BOOT_HDD_IMAGE") or (get_var('PATCHED_SYSTEM') and !get_var('ZDUP'))) {
    ...
````

Reply:
Ok, updated.